dvc/repo/reproduce.py

```diff
@@ -73,15 +73,15 @@ def reproduce(
     if not interactive:
         kwargs["interactive"] = self.config["core"].get("interactive", False)
 
-    active_graph = _get_active_graph(self.pipeline_graph)
+    active_graph = _get_active_graph(self.graph)
     active_pipelines = get_pipelines(active_graph)
 
-    path, name = parse_target(target)
+    path, name, tag = parse_target(target)
```

Comment:
btw might be a good time to throw out tags. But could be done separately too.

Reply:
I wanted to throw this away, but kept it for now. This should be my next PR, just want to get rid of it.